MixItUp.Base/MixItUp.Base.csproj

```diff
@@ -1,9 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
This was weird, but it was causing me problems if I didn't change it. I think it affects the build pipeline.